core/src/main/scala/org/http4s/syntax/KleisliSyntax.scala

```diff
@@ -19,16 +19,16 @@ trait KleisliSyntax {
 }
 
 trait KleisliSyntaxBinCompat0 {
```

Comment:
Binary compatibility resets in 1.0, so we don't need any hacks.  We should be able to just flatten this.